src/main/kotlin/no/nav/syfo/application/ForesporselService.kt

```diff
@@ -25,6 +26,12 @@ class ForesporselService(
             )
 
         val storedForesporsel = repository.createForesporsel(foresporsel)
+
+        try {
```

Comment:
I stedet for å try/catche her kunne man gjort logging i `onFailure` på resultatet man har i `journalforForesporsel`: så blir koden her litt cleanere.